release/ami/configure.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -xe
+
+sudo systemctl start docker.service
+sudo systemctl start containerd.service
+
+sudo systemctl enable docker.service
+sudo systemctl enable containerd.service
+
+docker run --rm --privileged aptman/qus -s -- --r -p
```

Comment:
This image has all the same static binaries, its just also a multi-arch one that I can install on arm and x64.

Reply:
It is also well-documented: https://dbhi.github.io/qus/intro.html